### ISSUE TITLE
RequireJS stuff.

### ISSUE THREAD

**Issue by aaronhourie** [ModernPayphones/pliny#2]
Okay, so, the problem is solved for plant being undefined. DNA is working in the same way. The problem now lies in being able to create a DNA object from the plant class.

<pre>var dna = new DNA()</pre>

This probably represents a fundamental misunderstanding in the way requireJS works that I was hoping you could maybe clear up.

**Comment by samdoiron** [comment]
First argument to define can be an array which takes filenames, just like in main.

**Comment by aaronhourie** [comment]
Right, thanks. I tried to find some documentation but I couldn't find anything related. Somewhere I can look for that stuff?

**Comment by samdoiron** [comment]
Only place I know is requirejs.com
